projects/plugins/jetpack/extensions/blocks/contact-form/editor.scss

```diff
@@ -37,19 +36,22 @@
 			}
 
 			&.jetpack-field__width-25 {
-				flex: 0 0 25%;
+				flex: 1 1 calc( 25% - calc(var(--wp--style--block-gap, 1.5rem) * 3) );
```

Comment:
Is there a reason we multiply the gap by `3` only for the 25% width field?

Reply:
Yes, a 25% means at most 4 inputs on the row, so that means 3 gaps between each input.

Reply:
Did some tests, and with the 3 multiplier, things look a bit weird:
![image](https://user-images.githubusercontent.com/7811225/196546554-21e7a3af-2fda-473a-aee8-e6edf2febc02.png)

Reply:
Nice! I'd swear I measured the same thing, but your sample is quite evident. So this is `calc( 25% - calc( var(--wp--style--block-gap) * 1 ) )` for the 25 wrapper? Works for 50 and 75 as well apparently?